scripts/src/report/report_info.py

```diff
@@ -0,0 +1,86 @@
+
+import os
+import sys
+import docker
+import logging
+import json
+import yaml
+try:
+    from yaml import CLoader as Loader, CDumper as Dumper
+except ImportError:
+    from yaml import Loader, Dumper
+
+REPORT_ANNOTATIONS = "annotations"
+REPORT_RESULTS = "results"
+REPORT_DIGESTS = "digests"
+REPORT_METADATA= "metadata"
+
+def getReportInfo(report_path,info_type,profile_type,profile_version):
```

Comment:
use the pythonic way to name methods: https://www.python.org/dev/peps/pep-0008/#method-names-and-instance-variables, I would run this through a linter like this one https://code.visualstudio.com/docs/python/linting

Reply:
space after comas in signature

Reply:
Fixed all my new functions.